src/Differ/MapDiffer.php

```diff
@@ -23,43 +25,17 @@
  */
 class MapDiffer implements Differ {
 
-	/**
-	 * @var bool
```

Comment:
I just tried to explain why these comments are always helpful. Just being consistent also makes code easier to read. Why do I have to search for that "bool" somewhere else, if it can be just right here?